api/schema/aws_account_resource.go

```diff
@@ -52,6 +52,9 @@ var (
 				"name": resource_schema.StringAttribute{
 					Description: "Display name for the AWS account.",
 					Required:    true,
+					Validators: []validator.String{
+						stringvalidator.LengthAtLeast(3),
+					},
 				},
 				"organization_master_account_id": StringResourceAttributeWithMode{
```

Comment:
Remove this `organization_master_account_id` attribute. We have decided to remove support for organizations.
And when we get around to supporting that, we shouldn't refer to the organization by master account ID.

Reply:
thats added to support existing contract. 
the ui calls depend on the existence of the master accountId to show in some ui endpoints.
else it fails. Hence added it.

Reply:
this field is needed by some ui endpoint to return properly as it checks this field
to identify existence of org or account.

Reply:
We will not support organization onboarding for now.
Please remove this attribute.